fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSlotReference.java

```diff
@@ -155,9 +157,15 @@ private <E extends Expression> List<E> bind(List<E> exprList, List<Plan> inputs,
     }
 
     private <E extends Expression> E bind(E expr, List<Plan> inputs, Plan plan, CascadesContext cascadesContext) {
-        List<Slot> boundedSlots = inputs.stream()
-                .flatMap(input -> input.getOutput().stream())
-                .collect(Collectors.toList());
+        List<NamedExpression> boundedSlots = inputs.stream()
+                .flatMap(input -> {
+                    Plan subPlan = ((GroupPlan) input).getGroup().getLogicalExpression().getPlan();
```

Comment:
i don't think this is right, because we could write sql like that
```sql
SELECT k1, COUNT(v1) FROM t1 GROUP BY k1 HAVING SUM(v2) > 10;
```
the right way to bind having is write another rule that split is predicates into three parts
1. AggregateFunctions **DO NOT** appear in child aggregate node
2. AggregateFunctions **DO** appear in child aggregate node
3. other expression

And then, 
1. push down the **first part** AggregateFunctions to aggregate node
2. replace **both first part and second part** AggregateFunctions by SlotReferences generated from aggregate node's output expression
3. if the first part is not empty, we need add a project node **on the top** of the filter node transformed from Having